src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+// Action
+const CHECK = 'CHECK';
+
```

Comment:
- [ ] Kindly create your actions using the _**duck pattern**_. It should look like this:

```
const CHECK = 'bookstore/categories/CHECK'
```

#### Kindly implement this change. You can read this [article](https://github.com/erikras/ducks-modular-redux) to get a better understanding of duck patterns.